enterprise/management/commands/create_enterprise_course_enrollments.py

```diff
@@ -108,7 +142,8 @@ def _fetch_course_enrollment_data(self, enterprise_customer_uuid):
                 ON ece.enterprise_customer_user_id = ecu.id
                 AND ece.course_id = sce.course_id
             WHERE
-                ece.id IS NULL
+                ecu.linked = true
```

Comment:
why we need this extra condition `linked = true`?

Reply:
Now, we don't delete the record of the model but we set linked=false. And we need only those records which are linked to enterprise_customers.